README.md

```diff
@@ -814,7 +814,7 @@ In this snippet, the `=>` arrow function has a comma sequence as the concise ret
 
 ## Rule: `"this"`
 
-The **proper-arrows**/*this* rule requires `=>` arrow functions to reference the `this` keyword. It also supports a `"never"` configuration mode, which reverses the rule and means that `=>` arrow functions must never use `this`.
+The **proper-arrows**/*this* rule requires `=>` arrow functions to reference the `this` keyword. It also supports a `"never"` configuration mode, which reverses the rule and means that `=>` arrow functions must never use `this` and `"never-global"` which restricts usage of global `this` in arrow functions.
```

Comment:
Instead of referring to a "global this", which is a bit confusing IMO, I think we should re-word this description as:

> ... It also supports a `"never"` configuration mode, which reverses the rule and means that `=>` arrow functions must never use `this`, as well as a "never-global" configuration mode which only disallows `this` usage in arrow functions that are in the global scope.

IOW, it's not a "global this", but a "global arrow function using a this". Does that make sense?

We should the reflect that same kind of wording in the other few places of this documentation.